docs/modules/ROOT/pages/tmail-backend/jmap-extensions/contactSupport.adoc

```diff
@@ -20,4 +20,6 @@ The associated object have the following fields:
 
 The support email address can be configured in the `jmap.properties` by setting the `support.mail.address` property.
 
+The support http link can be configured in the `jmap.properties` by setting the `support.httpLink` property.
```

Comment:
Is it allowed to get both support.mail.address and support.httpLink?

Please document it

Reply:
And it likely should not be.

Reply:
Yes, the current code allows for both configurations. 
IMO we shouldn't restrict it, as it depends on the admin's preferences. 

I will update the code so that if one configuration is not set, the corresponding JSON property will be absent instead of present with a NULL value.

Reply:
If both properties are present, then what should @hoangdat team display/do?

IMO we should be strict and allow just for one value

Please add the precomdition and test it.

Reply:
updated